utils/scanUtils.py

```diff
@@ -16,6 +16,7 @@ def dacScanAllLinks(args, calTree, vfatBoard):
 
     # Get the AMC
     amcBoard = vfatBoard.parentOH.parentAMC
+    nVFATs = vfatBoard.parentOH.nVFATs
```

Comment:
In many places you are using `vfatsPerGemVariant[args.gemType]` but here we are using the class data member.  Is there a reason for this inconsistency? e.g. are there some use cases where the class doesn't exist and so it would not be appropriate to take from this after initialization of a `HwVFAT` or `HwOptohybrid` object?  

Otherwise line 19 could probably be put in every script modified above and it might improve readability.  

I do not feel too strongly about this though.

Reply:
I've tried to use the `vfatsPerGemVariant[args.gemType]` where `HwVFAT` object does not exist, but might have overlooked in some places